content/reference/plugins/github_actions/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "GitHub Actions Plugin"
+description: "Learn how to configure the Gatling Enterprise GitHub Actions plugin and run your simulations."
+lead: "Run your Gatling Enterprise simulations from GitHub Actions."
+date: 2022-01-04T15:00:00+00:00
+lastmod: 2022-01-04T15:00:00+00:00
+weight: 23040
+---
+
+## Purpose of this GitHub Action
+
+This Action enables you to start a Gatling Enterprise simulation directly from your GitHub Actions workflows. This plugin links a workflow with one and only one Gatling Enterprise simulation.
+
+This plugin doesn't create a new Gatling Enterprise simulation, you have to create it using the Gatling Enterprise Dashboard before, or do it using the options provided by our build tools plugins (for [Maven](https://gatling.io/docs/gatling/reference/current/extensions/maven_plugin/#working-with-gatling-enterprise-cloud), [Gradle](https://gatling.io/docs/gatling/reference/current/extensions/gradle_plugin/#working-with-gatling-enterprise-cloud), [SBT](https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#working-with-gatling-enterprise-cloud)).
+
+Don't forget to check out [GitHub's official documentation](https://docs.github.com/en/actions) to learn how to write CI/CD workflows with GitHub Actions.
+
+## Releases
+
+You can check out the latest releases available [from the GitHub project](https://github.com/gatling/enterprise-action/releases). You should generally only specify the major version you want to use, currently `gatling/enterprise-action@v1`.
```

Comment:
Note: `v1` isn't actually available yet, will need to be released before we publish the documentation.